Content.Server/Guardian/GuardianSystem.cs

```diff
@@ -94,6 +98,9 @@ private void OnHostShutdown(EntityUid uid, GuardianHostComponent component, Comp
             if (component.HostedGuardian == null)
                 return;
 
+            if (HasComp<HandsComponent>(component.HostedGuardian.Value))
+                _bodySystem.GibBody(component.HostedGuardian.Value);
```

Comment:
but then dont i need to add 3 lines of code 1 for each item slot and hands rather than 1 line of code? i would essentially be copying code from gib to this to drop items.

Reply:
Why would it even check for handscomponent then?